hooks/playbooks/pre_test_ironic_flavor.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Create resources for running ironic scenario tests
```

Comment:
could this message be less generic, to differentiate its counterpart in the other hooks?

Reply:
Done